l10n_br_resource/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module extend core resource to create important brazilian informations. Define a Brazilian calendar and some tools to compute dates used in financial and payroll modules
```

Comment:
Thanks for the fixes. There are a few English errors here. I propose instead:
"This module extends the core resource module to add data for Brazil. It defines a Brazilian calendar and some tools to compute dates used in financial and payroll modules."